cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -905,13 +905,13 @@ public synchronized void executeDemoteProposals(Collection<ExecutionProposal> pr
   }
 
   /**
-   * Dynamically set the movement concurrency of the given type for all brokers.
+   * Dynamically set the per broker movement concurrency of the given type for all brokers or set the cluster concurrency.
    *
    * @param concurrency The maximum number of concurrent movements.
    * @param concurrencyType The type of concurrency for which the requested movement concurrency will be set.
    */
-  public synchronized void setExecutionConcurrencyForAllBrokers(Integer concurrency, ConcurrencyType concurrencyType) {
-    _executionTaskManager.getExecutionConcurrencyManager().setExecutionConcurrencyForAllBrokers(concurrency, concurrencyType);
+  public synchronized void setExecutionConcurrencyForAllBrokersOrCluster(Integer concurrency, ConcurrencyType concurrencyType) {
+    _executionTaskManager.getExecutionConcurrencyManager().setExecutionConcurrencyForAllBrokersOrCluster(concurrency, concurrencyType);
```

Comment:
Just curious, why not make 2 functions?
1. setExecutionConcurrencyForAllBrokers
2. setExecutionConcurrencyForCluster
That seems to be more clear and makes more sense to me

Reply:
btw since leadership is the only cluster level concurrency type, we can name it to setExecutionClusterLeadershipConcurrency

Reply:
I do not think there is big difference in the functionality between setting concurrency for all brokers and for cluster. And one function makes the code more concise. Otherwise, there would be repeated code in both functions like "sanityCheckRequestedConcurrency(concurrency, concurrencyType);" and " default:
        throw new IllegalArgumentException("Unsupported concurrency type " + concurrencyType + " is provided.");
    }
    refreshExecutionConcurrencySummary();"

Reply:
> I do not think there is big difference in the functionality between setting concurrency for all brokers and for cluster
Just curious, for "all brokers", we are setting a same concurrency value for each single broker, while for cluster concurrency, we are setting a single value for cluster level concurrency. Is that correct? In that case, I think they are still fundamentally different. E.g. In the concurrencyManager, we have separate functions for set broker concurrency and set cluster concurrency. Let me know if that makes sense.

Reply:
In ExecutionConcurrencyManager, there are two methods setExecutionConcurrencyForAllBrokersOrCluster and setExecutionConcurrencyForBroker, so for allBrokers and Cluster there is also one method. setExecutionConcurrencyForBroker is a separate method because the method take an additional parameter (brokerId).

Reply:
ack